x-pack/plugins/apm/server/feature.ts

```diff
@@ -60,3 +60,38 @@ export const APM_FEATURE = {
 
 export const APM_SERVICE_MAPS_FEATURE_NAME = 'APM service maps';
 export const APM_SERVICE_MAPS_LICENSE_TYPE = 'platinum';
+
+export const RUM_FEATURE = {
+  id: 'client_side_monitoring',
```

Comment:
**nit**: our convention for feature ids is camelCase:

```suggestion
  id: 'clientSideMonitoring',
```

**note**: Just so you're aware, once you commit to a feature id, you're not allowed to change it. This would break the security model. Just wanted to point this out in case you wanted to give this some more thought. I have no objections with your current name FWIW